README.md

```diff
@@ -142,6 +143,13 @@ Browserslist will take queries from tool option,
   than Microsoft Edge. Chinese QQ Browsers has more market share than Firefox
   and desktop Safari combined.
 
+## Why you need to call update-db script regularly?
+
+The `--update-db` flag will run a script which will modify your lockfile (e.g. `package-lock.json` or `yarn.lock`) to update the dependency `caniuse-lite` to the latest version.
+
+The reason you may need to run this script regularily has to do with NPM architecture and how lockfiles guarantee which version of a dependency your project is expecting. Unless you explicitely run `npm update`, your lockfile may have multiple references to `caniuse-lite`, each with different versions. 
```

Comment:
It will be better to explain why it’s happening as I explained in the issue's comment with an examples